libr/core/cmd_type.c

```diff
@@ -1232,6 +1233,25 @@ static int cmd_type(void *data, const char *input) {
 					sdb_close (db_tmp);
 					sdb_free (db_tmp);
 				}
+			}  else if (input[1] == 'e') { // toe
+				char *str = r_core_cmd_str (core , "tc");
+				char *tmp = r_core_editor (core, "*.h", str);
```

Comment:
Not sure if we want to edit all the includes

Reply:
No , afaik the `*.h` means to create a temporary file with ".h" extension !

Reply:
right :) i should pick a different char for that